test/float8/test_base.py

```diff
@@ -129,6 +136,111 @@ def test_copy_(self):
         fp8_b.copy_(fp8_a)
         torch.testing.assert_close(fp8_a._data, fp8_b._data)
 
+    @pytest.mark.parametrize("shape", [(8, 16), (4, 8, 16), (2, 4, 8, 16)])
+    @pytest.mark.parametrize("dim_name", ["first", "last"])
```

Comment:
nit: why not just `0`, `-1`

Reply:
will do, that didn't work in the original PR version but should work now